exercises/luhn/example.tt

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'luhn'
+
+# Common test data version: <%= abbreviated_commit_hash %>
+class LuhnTest < Minitest::Test<% test_cases.each do |test_case| %>
+  def <%= test_case.name %>
+    <%= test_case.skipped %>
+    <%= test_case.assertion %> <%= test_case.work_load %>
```

Comment:
`test_case.workload` should include the assertion.

So this line can be simplified to: `<%= test_case.workload %>`